packages/select/test/selectTests.tsx

```diff
@@ -119,6 +119,45 @@ describe("<Select>", () => {
 
     it("returns focus to focusable target after popover closed");
 
+    describe("dropdownRenderer", () => {
+        it("overrides default dropdown rendering", () => {
+            const customClass = "custom";
+            const wrapper = select({
+                dropdownRenderer: () => <ul className={customClass} />,
+            });
+            assert.lengthOf(wrapper.find(Menu), 0, "should not find Menu");
+            assert.lengthOf(wrapper.find(`ul.${customClass}`), 1, "should find custom class");
+            assert.equal(handlers.itemRenderer.callCount, 0, "itemRenderer should not be called");
+        });
+
+        it("renderItem calls itemRenderer", () => {
+            select({
+                dropdownRenderer: props => <ul>{props.items.map(props.renderItem)}</ul>,
+            });
+            assert.equal(handlers.itemRenderer.callCount, TOP_100_FILMS.length);
+        });
+
+        it("initialContent is only rendered if query is empty", () => {
```

Comment:
i think initial content should be handled in your custom `menuRenderer` implementation